client/src/pages/AddFood.js

```diff
@@ -42,27 +43,26 @@ function AddFood() {
   return (
     <div className="form">
       <Layout />
-      <Form onSubmit={onSubmit}>
+      <Form onSubmit={handleSubmit((data) => setData(JSON.stringify(data)))}>
         <h2>Add a New Food Item </h2>
         <label>Food Name</label>
         <input
           type="text"
-          name="food"
-          {...register("food")}
-          onChange={handleChange}
+          {...register("foodName")}
+          placeholder="Food name"
           value={food}
+          onChange={handleChange}
         />
-        {errors.foodName?.type === "required" && (
-          <p role="alert">Food name is required</p>
-        )}
-        <label>Food Description</label>
-        <textarea></textarea>
-        <label>Food author</label>
-        <select>
-          <option value="murial">murial</option>
-          <option value="brent">brent</option>
+        <label>Food Type</label>
+        <select {...register("category", { required: true })}>
+          <option value="Breakfast">Breakfast.</option>
+          <option value="Lunch">Lunch</option>
+          <option value="Dinner">Dinner</option>
         </select>
-        <Button variant="contained" onClick={handleSubmit}>
+        <label>Food Description</label>
+        <textarea {...register("aboutYou")} placeholder="About food" />
```

Comment:
Does `aboutYou` need to be in the schema for this to work as expected?

Reply:
No it doesn't ..updated it to match what is in the schema file